Rubberduck.Core/UI/SelectionChangeService.cs

```diff
@@ -33,47 +34,60 @@ public SelectionChangeService(IVBE vbe, IParseCoordinator parser)
             VBENativeServices.WindowFocusChange += OnVbeFocusChanged;
         }
         
-        private void OnVbeSelectionChanged(object sender, SelectionChangedEventArgs e)
+        private void OnVbeSelectionChanged(object sender, EventArgs e)
```

Comment:
why the despecialization? Hasn't been done for `OnVbeFocusChanged`

Reply:
@Vogel612 See [this comment](https://github.com/rubberduck-vba/Rubberduck/pull/4240#discussion_r206311809). I'm re-specializing this and I apparently forgot to stub the `SelectionChangedEventArgs`, so I caught that too.